bin/run_dsm.py

```diff
@@ -26,15 +25,34 @@ async def main(opts):
     await asyncio.gather(csc.done_task, go_to_enabled(csc.domain, opts))
 
 
-if __name__ == '__main__':
-    parser = argparse.ArgumentParser(description="Startup the DSM CSC. Default mode is real time telemetry.")
-    parser.add_argument("-i", "--index", type=int, default=1,
-                        help="SAL index; use the default value unless you sure you know what you are doing")
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Startup the DSM CSC. Default mode is real time telemetry."
+    )
+    parser.add_argument(
+        "-i",
+        "--index",
+        type=int,
+        default=1,
+        help="SAL index; use the default value unless you sure you know what you are doing",
+    )
     group = parser.add_mutually_exclusive_group()
-    group.add_argument("-f", "--fast", dest="mode", action="store_const", const=1,
-                       help="Run CSC in fast (1 sec) telemetry simulation mode")
-    group.add_argument("-s", "--slow", dest="mode", action="store_const", const=2,
-                       help="Run CSC in slow (30 sec) telemetry simulation mode")
+    group.add_argument(
+        "-f",
+        "--fast",
+        dest="mode",
+        action="store_const",
+        const=1,
+        help="Run CSC in fast (1 sec) telemetry simulation mode",
+    )
+    group.add_argument(
+        "-s",
+        "--slow",
+        dest="mode",
+        action="store_const",
+        const=2,
+        help="Run CSC in slow (30 sec) telemetry simulation mode",
+    )
```

Comment:
I'm a bit worried about folks thinking -s means "simulate". It's a bit more typing, but you might consider `-s`/`--simulate` with a required value "fast" or "slow". Or pick one to be the default.

Reply:
Since I don't use the amain form of executor since it does not play well with the auto transition function, the --simulate flag doesn't show up if you run --help on the scripts.

Reply:
I am suggesting changing your API to match the standard CSC API: --simulate with a value. I agree that you will have to implement it yourself, but it's a minor change to your existing code.

Reply:
Done.